documentation/practices.md

```diff
@@ -52,15 +52,13 @@ Catalog images and site content images can be uploaded to WCH using the wchtools
 The React PWA Reference Storefront is pre-configured for integration with the Google Analytics enhanced e-commerce plugin. 
 
 ### Route/Page Views
-The handlers for integrating with Google Analytics allow for routes, which are pre-configured in the `App.js` file, to be logged as pageviews in the sites real-time traffic data. Further configuration in Google Analytics will enable measurements for conversion rates based on the pages viewed in the web traffic. 
-`Analytics.js` contains the functions used for logging page views with ReactGA (React library for invoking Google Analytics functions), and functions for invoking the enhanced e-commerce capabilities.
+In the `App.js` file, integration handlers for Google Analytics are pre-configured to allow for routes to be logged as pageviews in a sites real-time traffic data. Any further measurements for conversion rates based on the page views in the web traffic must be configured in Google Analytics.
+The `Analytics.js` file contains the functions used for logging page views with the React library for invoking Google Analytics functions (ReactGA), and enhanced e-commerce capabilities.
 
 ### Checkout Flow
-The checkout flow is pre-configured to log purchases through Google Analytics' basic e-commerce capabilities. It will submit transaction/purchase amounts along with cart information where it can be further analyzed in Google Analytics under **Conversions > Ecommerce**. The product and cart pages are configured to submit information when a product is viewed, added to cart, or removed from the cart.
+The check-out flow is pre-configured to log purchases through Google Analytics' basic e-commerce capabilities. When a customer completes a transaction, Google Analytics analyzes the logged transaction/purchase amounts and cart information under **Conversions > Ecommerce**. When a customer views a product, adds a product to the cart, or removes a product from the cart, the product and cart pages submit this information for further analysis. 
```

Comment:
This has become far too high-level IMO. We actually do submit transaction/purchase amounts along with cart information. This is a separate action which is not part of the action for submitting transaction information to Cortex. Let me know if this is unclear and we can discuss more.

Reply:
This one I'm struggling with a bit more.  I'm not sure how this sounds @shaunmaharaj @sminnath 
The transaction and purchase amounts, along with the cart information, are submitted under **Conversions > Ecommerce**, where Google Analytics analyzes the information.

Reply:
Yes, but they're not submitted a part of the transaction information sent to Cortex. I agree this needs to be revised, but we can't give the impression that "when a customer completes a transaction, Google Analytics analyzes the information". That's true at a high-level, but not technically the case.

Reply:
How about this? I can also break down the second sentence as well, I feel it's long.

The check-out flow is pre-configured to log purchases through Google Analytics' basic e-commerce capabilities. Upon successful completion a transaction, the storefront sends another request to submit the transaction and purchase amounts along with cart information where it can be further analyzed in Google Analytics under **Conversions > Ecommerce**.